Azure-Samples/chat-with-your-data-solution-accelerator
# fix: font size changes reverted and removed extra padding

## Purpose
Revert the font size changes and removed extra padding

## Does this introduce a breaking change?





## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
open cwyd chatbot page and check the font size and spacing 

![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/168007985/ec5e2b7c-eef9-45b2-ade3-8acf59ede12e)
